src/theme/components/multiSelect.ts

```diff
@@ -0,0 +1,97 @@
+const multiSelectWidths = {
+  default: {
+    width: {
+      base: "100%",
+      md: "250px",
+    },
+  },
+  fitContent: {
+    width: {
+      base: "100%",
+      md: "fit-content",
+    },
+    minWidth: {
+      md: "175px",
+    },
+  },
+  full: {
+    width: "100%",
+  },
+};
+
+const MultiSelect = {
+  parts: [
+    "menuButton",
+    "menuContainer",
+    "menu",
+    "menuChildren",
+    "selectedItemsCountButton",
+  ],
+  baseStyle: ({ width = "default", isBlockElement }) => ({
+    ...multiSelectWidths[width],
+    position: "relative",
+    margin: {
+      base: ".75rem 0",
+      md: "0 2em 0 0",
+    },
+    menuButton: {
+      width: "100%",
+      justifyContent: "space-between",
+    },
+    menuContainer: {
+      position: {
+        md: isBlockElement ? "relative" : "absolute",
+      },
+      zIndex: isBlockElement ? 0 : 1,
+      ...(width === "fitContent" && {
+        minWidth: {
+          md: "275px",
+        },
+        maxWidth: {
+          md: "375px",
+        },
+      }),
```

Comment:
I'm not sure about this so I'll defer to @bigfishdesign13 , but for "fitContent", shouldn't the width also be set to this value? Such as `width: "fit-content",`.

Reply:
This we'll have to keep as is I think. It sets "fit-content" on the multiselect container div, and then the max/min allows for the menu to be wider than the menu button. Example here:

<img width="833" alt="multiselect-fitcontent" src="https://user-images.githubusercontent.com/62392174/171921734-b0c9e9ef-dc30-4953-b5f5-e53df6315e2f.png">

 If this was "default" (button and menu same width), then this would be too narrow.